src/components/site-header/_site-header-language-switcher.scss

```diff
@@ -269,47 +300,63 @@ $language-list: null !default;
   // Horizontal sections
   .ecl-site-header__language-content:not(
       .ecl-site-header__language-content--stack
+    ),
+  .ecl-site-header__custom-action-content:not(
```

Comment:
there are things not needed for the custom action in this css file; the language switcher has some specific display in column, but that's not the case here

Reply:
yes, i cleaned up